docs/requirements.txt

```diff
@@ -1,6 +1,6 @@
-Sphinx==1.8.1 \
-    --hash=sha256:652eb8c566f18823a022bb4b6dbc868d366df332a11a0226b5bc3a798a479f17 \
-    --hash=sha256:d222626d8356de702431e813a05c68a35967e3d66c6cd1c2c89539bb179a7464
+Sphinx==1.8.3 \
+    --hash=sha256:429e3172466df289f0f742471d7e30ba3ee11f3b5aecd9a840480d03f14bcfe5 \
+    --hash=sha256:c4cb17ba44acffae3d3209646b6baec1e215cad3065e852c68cc569d4df1b9f8
```

Comment:
I'm not sure we need this and related requirements in this file. We have a different requirements set for docs-building.